crates/nu-parser/fuzz/gather_seeds.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Check if 'seeds' directory exists. If not, create one.
+if [ ! -d "seeds" ]; then
+    mkdir seeds
+fi
+
+# Find all "*.nu" files from '../..' excluding the 'seeds' directory and copy them into 'seeds' directory.
+find ../../.. -type f -name "*.nu" -exec cp {} seeds/ \;
+
```

Comment:
don't know if `/bin/bash` as a locatoin is guaranteed :P

Should this step be mentioned in the Readme?

If you want a larger corpus of data, one could scrape the https://github.com/nushell/nu_scripts repo. Don't know where there are diminishing returns with regards to entropy.

Reply:
> don't know if /bin/bash as a locatoin is guaranteed :P

Now I switched to nuscript.

> If you want a larger corpus of data, one could scrape the https://github.com/nushell/nu_scripts repo. Don't know where there are diminishing returns with regards to entropy.

Yes usually more and diverse sets of initial inputs is always good. but for now i will keep this as default, and the fuzzing-practitioner can change it as they want/